wger/settings_global.py

```diff
@@ -195,6 +213,15 @@
 LOGIN_URL = '/user/login'
 LOGIN_REDIRECT_URL = '/'
 
+# Twitter social-auth settings
+SOCIAL_AUTH_TWITTER_KEY = 'IxPnp6wm11EinVYQNqcEa9poZ'
+SOCIAL_AUTH_TWITTER_SECRET = 'zuEz1B4pIwF1SSYAt6tZNxiOE8EvI2vzZYB1Xyu8av2RavvjKX'
+
+SOCIAL_AUTH_FACEBOOK_KEY = '451230318647842'
+SOCIAL_AUTH_FACEBOOK_SECRET = '7919df747bfa89b98a7be5cbfb3d744e'
+
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY ='412817226779-firpa2n05rtohpt2non2rv727l1ihb5j.apps.googleusercontent.com'
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = 'XKOQj1oPnwGLN35t9-PikNTu'
 
```

Comment:
Kindly make all these keys environment variables

Reply:
I have set the keys as environment variables